components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/matcher/UrlMatcher.kt

```diff
@@ -23,7 +23,7 @@ class UrlMatcher {
     private val whiteList: WhiteList?
     private val previouslyMatched = HashSet<String>()
     private val previouslyUnmatched = HashSet<String>()
-    private var blockWebfonts = true
+    private var blockWebfonts = false
```

Comment:
I don't see how `DefaultSettings(webFontsEnabled = ...)` will lead to this value being true/false? Isn't that part missing here?

Reply:
@pocmo we didn't need that as we've never matched against that URL matching category WEBFONT, but checked for webfonts explicitly:

https://github.com/mozilla-mobile/android-components/blob/master/components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineView.kt#L162

Alternatively, we could also add a parameter to `matches` to only check certain (URL matching) categories, but it made sense to me to keep this separate, now that it's not a tracking protection category.

Reply:
Since it was confusing to get rid of the TrackingProtectionPolicy.WEBFONTS but keep the UrlMatcherCategory WEBFONTS, I got rid of both now.

I think this is easier to read. We can either configure TrackingProtectionPolicies which directly map to UrlMatcher categories, our we can enable/disable webfonts using the engine setting (same logic as on master except that we no longer block web fonts by default/mistake).